src/schema/v2/partner/partner.ts

```diff
@@ -576,6 +576,11 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ default_profile_public }) => default_profile_public,
       },
+      isInquireable: {
+        type: GraphQLBoolean,
```

Comment:
**suggestion / non-blocking**: We generally prefer to make resolver types non-nullable if possible although I notice that many types in this Partner type are already nullable. Since the definition of `inquireable?` in Gravity ([source](https://github.com/artsy/gravity/blob/9fed3fe7e8cb20e50fb5bf268f9c456d149038aa/app/models/domain/partner.rb#L356-L358) 🔒) always returns a boolean and this JSON property should always be present in the JSON response, what do you think of upgrading this field to non-null?

```ruby
isInquireable: {
  type: new GraphQLNonNull(GraphQLBoolean),
```

Reply:
that makes total sense, updating